src/inc/gcinfotypes.h

```diff
@@ -262,6 +262,20 @@ inline bool IsStructReturnKind(ReturnKind returnKind)
     return returnKind > 3;
 }
 
+inline bool IsScalarReturnKind(ReturnKind returnKind)
+{
+    return (returnKind == RT_Scalar)
+#ifdef _TARGET_X86_
+        || (returnKind == RT_Float)
+#endif // _TARGET_X86_
+        ;
+}
+
+inline bool IsPointerReturnKind(ReturnKind returnKind)
+{
+    return !IsScalarReturnKind(returnKind);
```

Comment:
Do you want to assert that it is not `RT_Unset` or `RT_Illegal`?

Reply:
Thanks, done.